Module/nss/openstore001.nss

```diff
@@ -0,0 +1,13 @@
+#include "nw_i0_plot"
+void main()
```

Comment:
No NWScripts please. If you need a store, use the action-open-store snippet detailed here: https://wiki.starwarsnwn.com/Development/snippets

All new code must be in C#.